### PR TITLE
chore(docs): improve `LATEST ON` reference

### PR DESCRIPTION
**Pull request by amyshwang**
1. Remove `LATEST BY` info
2. Simplify wording

### CODE REVIEW

File: docs/reference/sql/latest-on.md [hunk 1]
```diff
@@ -6,11 +6,8 @@ description:
   illustration.
 ---
 
-For scenarios where multiple time series are stored in the same table, it is
-relatively difficult to identify the latest items of these time series with
-standard SQL syntax. QuestDB introduces `LATEST ON` clause for a
-[SELECT statement](/docs/reference/sql/select) to remove boilerplate
-clutter and splice the table with relative ease.
+Retrieves the latest entry by timestamp for a given key or combination of
+keys,for scenarios where multiple time series are stored in the same table.
```
Comment (bziobrowski) [inline docs/reference/sql/latest-on.md:10]: Seems like a whitespace is missing between "keys,for" .

File: docs/reference/sql/latest-on.md [hunk 2]
```diff
@@ -59,32 +66,6 @@ used in the `LATEST ON` part of the clause stands for the designated timestamp
 column for the table. This allows the database to find the latest value within
 each time series.
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/reference/sql/latest-on.md:68]: Adding L62-87 back.

[the remark this answers is not in the document]
Comment (bziobrowski) [reply docs/reference/sql/latest-on.md]: 👍

**Review by bziobrowski — APPROVED** [review]
Looks good to me .

**Pull request merged**